src/domains/profile/pages/ImportProfile/ProfileFormStep.tsx

```diff
@@ -156,14 +163,7 @@ const CreateProfileForm = ({
 							<>
 								<FormField name="password">
 									<FormLabel label={t("SETTINGS.GENERAL.PERSONAL.PASSWORD")} optional />
```

Comment:
```suggestion
									<FormLabel label={t("SETTINGS.GENERAL.PERSONAL.PASSWORD")} optional={!password && !confirmPassword}/>
```
to matching behavior on [the create profile page](https://github.com/PayvoHQ/wallet/blob/76693753bcd0d2861668a19e3f965e44caf0631d/src/domains/profile/pages/CreateProfile/CreateProfile.tsx#L144), i think you can change optional label with this and [also here](https://github.com/PayvoHQ/wallet/pull/115/files#diff-85456e7103f4cb8ada5cb9f278efeffb4f0f2662987e6ea9e8f8fe1b49c4e4a6R172)
as you know for this, it has to watch `confirmPassword`, then you can remove `getValues` [here](https://github.com/PayvoHQ/wallet/pull/115/files#diff-85456e7103f4cb8ada5cb9f278efeffb4f0f2662987e6ea9e8f8fe1b49c4e4a6R89)

Reply:
done!